src/annotator/components/NotebookModal.tsx

```diff
@@ -107,14 +116,18 @@ export default function NotebookModal({
 
   return (
     <div
-      className={classnames(
-        'fixed z-max top-0 left-0 right-0 bottom-0 p-3 bg-black/50',
-        { hidden: isHidden },
-      )}
+      className={classnames({ hidden: isHidden })}
```

Comment:
I considered moving this to the `ModalDialog` classes, but since we are not removing it from the DOM but just hiding it, that left the backdrop still visible.